src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/integration/CourtIntegrationTest.kt

```diff
@@ -32,10 +34,27 @@ class CourtIntegrationTest : IntegrationTest() {
   @Autowired
   lateinit var objectMapper: ObjectMapper
 
+  @Autowired
+  lateinit var telemetryClient: TelemetryClient
```

Comment:
would it not be better to mock this out?

Reply:
Yes. Done.